src/pudl/metadata/resources/eia923.py

```diff
@@ -2,6 +2,26 @@
 
 from typing import Any
 
+from pudl.metadata.resources.eia import inherits_harvested_values_details
+
+
+def merge_descriptions(left: dict[str, Any], right: dict[str, Any]) -> dict[str, Any]:
+    """Merge two description dictionaries."""
```

Comment:
we need this because 923 has a buuuuuunch of description dictionaries that are shared between core and out, but only the out layer tables need the harvest warnings

Reply:
I guess this could live somewhere more central but 923 is the only source that had this particular problem